core/html-matrix-build.js

```diff
@@ -67,13 +68,7 @@ class ManifestObjects {
         if (version.trim().length < 1) {
           throw new Error('common-version may not be empty.');
         }
-        if (commonVersion) {
-          if (commonVersion !== version) {
-            throw new Error(`common-version '${version}' must match '${commonVersion}' from previously processed manifests.`);
-          }
-        } else {
-          commonVersion = version;
-        }
+        commonVersion = getHigherVersion(commonVersion, version);
```

Comment:
Probably not worth worrying about now, but worth considering still throwing an error if two different _major_ versions are used together, since we likely wouldn't be able to reconcile whatever breaking changes would happen if/when we release features 2.0